### ISSUE TITLE
Can library handle #REF errors in excel and pass on value as #REF only

### ISSUE THREAD

**Issue by Avik1993** [vinci1it2000/formulas#46]
**Describe the bug**
I have a big file with numerous references to other files and some links as well for data, when I try to read that using `formulas` library it fails with :
```
FormulaError: ('Not a valid formula:\n%s', '=[1]Sheet3!#REF!')
```

Even though I don't have any Sheet3 in the excel, I still see this error.
Can library by-pass such occurrences and put the value as #REF itself on calculation?

**To Reproduce**
Read any excel file with incorrect formula.

**Expected behavior**
Would like if the library can imitate Excel behavior and just say #REF as the value of such cells.

**Comment by mishu-** [comment]
+1

**Comment by vinci1it2000** [comment]
This issue has been fixed with 52ffdf0aee6896287d414359a900117a4199b9d9. Temporarily, you can use my dev.
